bin/cassandra

```diff
@@ -132,21 +132,21 @@ find_library()
     pattern=$1
     path=$(echo ${2} | tr ":" " ")
 
-    find $path -regex "$pattern" -print 2>/dev/null | head -n 1
+    find $path -regex "$pattern" -print 2>/dev/null | sort -r | uniq | head -n 1
 }
 case "`uname -s`" in
     Linux)
         if [ -z $CASSANDRA_LIBJEMALLOC ] ; then
             which ldconfig > /dev/null 2>&1
-            if [ $? = 0 ] ; then
+            if [ ! -f /etc/lsb-release ] ; then
```

Comment:
The above check makes sure the command exists, would keep it unless you found a bug.